IPython/core/tests/test_interactiveshell.py

```diff
@@ -171,9 +171,9 @@ def test_can_pickle(self):
                      "    def __init__(self,x=[]):\n"
                      "        list.__init__(self,x)"))
         ip.run_cell("w=Mylist([1,2,3])")
-        
```

Comment:
Could we ask you to redo this without removing all the trailing whitespace from the files? Sorry, but it messes up git blame and can cause unnecessary merge conflicts.

If you're using Atom, which strips trailing whitespace by default, you can disable that in the settings of the 'whitespace' package.